src/components/FormType.tsx

```diff
@@ -38,11 +40,17 @@ const CalendarFormType = () => (
 const GitHubFormType = ({ modals }: GitHubFormTypeProps) => {
 
   const { setCreateModalState, setRulesetModalState } = modals
+  const { data: session } = useSession()
+  
+  // show some ui error handling
```

Comment:
are these comments TODOs?

Reply:
yeah the comment is a future todo. there could be a chance where the user tried to create an automation but the access token is undefined so it's impossible to create the hook. If the session is undefined or if session.user.accessToken is undefined (todo), then tell the user to sign out and sign back in

Reply:
if it's a TODO best to mark it as such. internally we like to file tasks for each TODO and add the task number to the comment, but for your case it's a bit overkill to file Github issues for each TODO

Reply:
this is just a stylistic preference but I usually just prefix the comment with TODO like

```
// TODO: show some ui error handling
```

that way people know that it's a TODO and not a comment describing the actual code. otherwise it's confusing because it looks like any other comment except it doesn't match what the code actually does